openadapt/adapters/ultralytics.py

```diff
@@ -135,8 +137,29 @@ def do_fastsam(
         logger.info(f"{annotation.path=}")
         segmented_image_path = Path(tmp_dir) / result_name
         segmented_image = Image.open(segmented_image_path)
-        os.remove(segmented_image_path)
-
+        # os.remove(segmented_image_path)
```

Comment:
What do you think about removing this?

Reply:
I've added this in a while loop where I attempt to delete file with retries and delay

Reply:
Right, so why keep this commented out line?

Reply:
removed in `436af727ccc966d15ecd33ed13af1a05d2b32e20`